components/LeftImageTextLayout.jsx

```diff
@@ -21,32 +41,72 @@ export const LeftImageTextLayout = ({
       </Title>
     </TitleSection>
     <InfoSection>
-      <Graphic src={graphic} width={300} height={240} />
-      <Description>{descriptionText}</Description>
+      <Graphic src={graphic} width={imageWidth} height={imageHeight} />
+      <Description isAlignedTop={isAlignedTop} imageHeight={imageHeight}>
+        <div>
+          {descriptions.map((paragraph) => (
+            <Text key={paragraph}>{paragraph}</Text>
+          ))}
+        </div>
+        {!!buttons && (
+          <ButtonSection>
+            {buttons.map((button) => (
+              <Link key={button.text} href={button.link}>
```

Comment:
Also I don't think this key is good enough. Keys have to be unique and I feel like the button text can be repeated. So is there any other way?

Reply:
Yep, I'll change to button link (at least in a layout there shouldn't be repeats)